dev/webpack.config.js

```diff
@@ -0,0 +1,160 @@
+/* eslint import/no-commonjs: 0 */
+const path = require('path');
+const without = require('lodash/without');
+
+const webpack = require('webpack');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const HtmlWebpackPluging = require('html-webpack-plugin');
+const BabiliPlugin = require('babili-webpack-plugin');
+const AutoDllPlugin = require('autodll-webpack-plugin');
+const HappyPack = require('happypack');
+
+const __DEV__ = process.env.NODE_ENV === 'development';
+const clean = arr => arr.filter(item => item !== false);
+
+module.exports = {
+  devtool: __DEV__ ? 'cheap-eval-source-map' : 'cheap-module-source-map',
```

Comment:
* with `source-map` you get the line and the col of the exception
* with `cheap-module-source-map` you only get the line